compiler/tflite2circle/src/CircleModel.cpp

```diff
@@ -412,11 +435,79 @@ void CircleModel::model_build(void) const
   circle::FinishModelBuffer(*_fb, model);
 }
 
+void CircleModel::finalize(void)
+{
+  if (_buffer_data_map.empty())
+    return;
+
+  auto align16 = [](size_t &v) {
+    while (v % 16 != 0)
```

Comment:
Just curiosity, why is it 16 instead of 8?

Reply:
tflite comment says each buffer data block must be aligned in 16 bytes.
I don't know why but maybe cause of cache alignment or memory access aligments.